final_project/dags/process_sales.py

```diff
@@ -0,0 +1,67 @@
+from datetime import datetime
+
+from airflow import DAG
+from airflow.providers.google.cloud.operators.bigquery import BigQueryInsertJobOperator
+
+from table_defs.sales_csv import sales_csv
+
+DEFAULT_ARGS = {
+    'depends_on_past': True,
+    'retries': 3,
+    'retry_delay': 5,
+}
+
+with DAG(
+        dag_id='process_sales',
+        description="Ingest and process sales data",
+        start_date=datetime(2022, 9, 1),
+        end_date=datetime(2022, 9, 30),
+        schedule_interval="@daily",
+        catchup=True,
+        tags=['sales'],
+        default_args=DEFAULT_ARGS,
+        max_active_runs=1
+
+) as dag:
+
+    dag.doc_md = __doc__
+
+    transfer_from_data_lake_to_raw = BigQueryInsertJobOperator(
+        task_id='transfer_from_data_lake_to_raw',
+        dag=dag,
+        gcp_conn_id='gcp-conn',
+        location='us-east1',
+        project_id='de2022-stanislav-rezen',
+        configuration={
+            "query": {
+                "query": "{% include 'sql/sales_transfer_from_data_lake_raw_to_bronze.sql' %}",
```

Comment:
I would suggest using subdirectories for each source: `sql/sales/`, `sql/customers/` etc
Keeping all files in one directory will create a mess